src/loginServices.js

```diff
@@ -58,8 +58,10 @@ export const supportedLocales = [
   'fr-FR',
   'de-DE',
   'he',
+  'hi_IN',
```

Comment:
I think this should be `hi-IN`.

Reply:
Ack — you're right.  Thank you.  I'm presuming the language list in ui-developer is also needed since that list is still there.

Reply:
I forgot ui-developer still had its own list; I just pushed up a PR [to remove its local copy](https://github.com/folio-org/ui-developer/pull/164). We already [factored the local copy out](https://github.com/folio-org/ui-tenant-settings/pull/140) of ui-tenant-settings.

Reply:
Makes sense—thank you so much!  I pulled [UID-28](https://issues.folio.org/browse/UID-28) into Core F Sprint 98 along with STCOR-458.  I'll update and close them.